### PR TITLE
[#266] User profile view (connects #266)

### PR DESCRIPTION
**Pull request by Rupesh87**

### CODE REVIEW

File: src/app/users/user-profile/user-profile.component.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<h3 i18n>Member Profile</h3>
+<form [formGroup]="editForm" (ngSubmit)="onSubmit()">
+  <mat-grid-list cols="3">
```
Comment (paulbert) [inline src/app/users/user-profile/user-profile.component.html]: Please use `<div>` tags instead.  The `mat-grid-list` isn't doing anything without `mat-grid-tile` as child components, and the layout will not be good without doing a lot of work to tweak the grid attributes.

File: src/app/users/user-profile/user-profile.component.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    <mat-form-field>
+      <input matInput i18n-placeholder placeholder="First Name" formControlName="firstName" required>
```
Comment (paulbert) [inline src/app/users/user-profile/user-profile.component.html]: You should use `planet-form-error-messages` here like in the `courses-add` component.

Comment (nirojdyola) [reply src/app/users/user-profile/user-profile.component.html]: @Rupesh87  Could you please use error message ?

**Comment by lmmrssa** [comment]
screen shot 2018-01-02 at 10 11 09
Here it seems user can repeat for admin and user.
Username needs to be unique so can this be checked when we create user

**Comment by paulbert** [comment]
Made a few updates.  I didn't foresee us uploading attachments for databases other than `resources`, so my mistake not adding the `prepAttachment` function to a shared service.

I don't want to cause more merge conflicts by fixing the `resources-add.component` to use that shared function, but we should update that once we've merged most of the backlog.

File: src/app/home/home.module.ts [hunk 3]
```diff
@@ -33,10 +31,9 @@ import { ManagerDashboardComponent } from '../manager-dashboard/manager-dashboar
     HomeComponent,
     DashboardComponent,
     NavigationComponent,
-    UsersComponent,
     CommunityComponent,
     NationComponent,
-    ManagerDashboardComponent
+    ManagerDashboardComponent,
```
Comment (lmmrssa) [inline src/app/home/home.module.ts]: why is there ',' at the end of array list

**Pull request merged**